.taskcluster.yml

```diff
@@ -0,0 +1,49 @@
+version: 1
+policy:
+  pullRequests: public
+tasks:
+  $let:
+    user: ${event.sender.login}
+
+    head_branch:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.head.ref}
+      else:
+        $if: 'tasks_for == "github-push"'
+        then: ${event.ref}
+        else: ${event.release.target_commitish}
+
+    head_rev:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.head.sha}
+      else:
+        $if: 'tasks_for == "github-push"'
+        then: ${event.after}
+        else: ${event.release.tag_name}
+
+    repository:
+      $if: 'tasks_for == "github-pull-request"'
+      then: ${event.pull_request.head.repo.html_url}
+      else: ${event.repository.html_url}
+  in:
+    - taskId: {$eval: as_slugid("tests_task")}
+      created: {$fromNow: ''}
+      deadline: {$fromNow: '1 hour'}
+      provisionerId: proj-relman
+      workerType: ci
+      payload:
+        maxRunTime: 3600
+        image: python:3.7
+        command:
+          - "/bin/bash"
+          - "-lcx"
+          - "git clone --quiet ${repository} &&
+               cd relman-auto-nag &&
+               git -c advice.detachedHead=false checkout ${head_rev} &&
+               travis_retry pip install --quiet -r requirements.txt &&
+               pre-commit run --all-files --show-diff-on-failure
+               tox -e $TOX_ENV"
```

Comment:
These are still not the same commands we were running on Travis.
E.g. we were installing requirements-test.txt, not requirements.txt.